packages/core/src/components/TextField/TextField.tsx

```diff
@@ -49,39 +48,100 @@ export type TextFieldProps = MaterialTextFieldProps & {
   'data-testid'?: string;
 };
 
-export default function TextField(props: TextFieldProps): JSX.Element {
-  const { name, onChange: baseOnChange, 'data-testid': dataTestid, inputProps, ...rest } = props;
-  const { control, errors } = useFormContext();
-  const errorMessage = get(errors, name);
+/**
+ * Convert original form rules to `react-hook-form` compatible rules.
+ */
+function normalizeRules<T>(rules?: ReactRules<T>): ControllerProps<ReactElement<T>>['rules']|undefined {
+  if(!rules){
+    return undefined;
+  }
+  
+  const newRules: ControllerProps<ReactElement<T>>['rules'] = {};
+  
+  if(typeof rules.min === 'object' && rules.min.value !== undefined){
```

Comment:
nit: I'm guessing that prettier hasn't been used to format these changes. Can you re-save the files in VSCode with the latest ESLint/prettier extensions installed?

Reply:
So VSCode is preferred for developing the GUI...
I personally love JetBrains' WebStorm for Typescript/Javascript developing but I'll do with VSCode from now on.